contrib/charts/pebble/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: pebble
+version: 0.1.0
```

Comment:
Can't see a 0.1 in https://github.com/letsencrypt/pebble/releases

Reply:
Chart version != appVersion, and given this is a chart used just for testing I've not bothered with proper versioning. The image itself is also manually pushed by myself, and is a snapshot from pebble's master branch.